pixi/backend/api.js

```diff
@@ -97,6 +127,7 @@ api.get('/lint', async (request, response) => {
 
   const fetchUrl = request.query.url;
   try {
+    await logAnalytics(fetchUrl);
```

Comment:
Why block on logging?

Reply:
thanks! vestigial code, removed the await in the last update